Task_TheFlexGame/index.css

```diff
@@ -1 +1,59 @@
-/* Write code below for FLEX GAME TASK */
+<html>
+    <head>
+        <title>The Flex Game</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <style>
+   
+            .dash-container{
+            display:grid;
+             height:100%;
+             width:100%;
+            grid-template-columns:20% 80%;
+            grid-template-rows:20% auto;
+            }
+           .dash-item{ 

```

Comment:
The color value should be in hex format (#FFFFFF) instead of #FFFF for better clarity.